pykern/pkunit.py

```diff
@@ -184,10 +184,14 @@ def file_eq(expect_path, *args, **kwargs):
     """
     import pykern.pkjson
     import pykern.pkconst
+    from pykern.pkdebug import pkdp
 
     def _convert_xlsx(expect_path, actual_path):
```

Comment:
These functions are getting complicated enough that we might as well make this a private class with methods that use instance attributes.